yoti-sdk-impl/src/main/java/com/yoti/api/client/shareurl/policy/SimpleWantedAttribute.java

```diff
@@ -29,6 +29,10 @@
     private final List<Constraint> constraints;
 
     SimpleWantedAttribute(String name, String derivation, boolean optional, Boolean acceptSelfAsserted, List<Constraint> constraints) {
+        if (name == "") {
+            throw new IllegalArgumentException("Attribute name missing");
+        }
```

Comment:
I suggest we use the Validation module we have that gets used in the rest of the system.  Suggested change:

```java
Validation.notNullOrEmpty(name, "name");
```

Can we also add in a test to ensure the exception is thrown if a name isn't supplied?